torch/lib/THCUNN/generic/PReLU.cu

```diff
@@ -23,11 +23,13 @@ void THNN_(PReLU_updateOutput)(
     input = THCTensor_(newContiguous)(state, input);
 
     int n = THCTensor_(nElement)(state, input);
+    if (input->size[ndim > 1] != nOutputPlane)
+      THError("wrong number of input planes");
```

Comment:
it's very helpful to report the number of input planes. We have been slowly and proactively changing error messages to do this, so this is a good time to do:
```
THError("wrong number of input planes. Expected %d but got %d.", input->size[ndim > 1], nOutputPlane);
```